trace_decoder/tests/trace_decoder_tests.rs

```diff
@@ -0,0 +1,272 @@
+//! These tests rely on the jerigon and cdk erigon witness files as input
+//! To add a new block or range of blocks as test input, do the following:
+//! 1. Run rpc tool to fetch block witness:
+//!```
+//!     cargo run --bin rpc fetch --rpc-url <node_rpc_endpoint> --start-block <start> --end-block <end> ./b<number>_<network>.json
```

Comment:
nit: maybe we should just refer to `zero_bin/README` instead for the command, as otherwise I'm afraid any CLI change would be forgotten here.

Reply:
@Nashtare This is tweak of test data itself for trace decoder, not sure if it should go the user oriented zero bin readme?

Reply:
My point was just that if we ever change the CLI on zero-bin, this comment will most likely be forgotten and become outdated.